### PR TITLE
Updated usage docu

### LINKED ISSUE #158: Do another lookover of code and docu
### Description of feature

Maybe the script can be shortened further? E.g. only one call of gost

### PR DESCRIPTION
**Pull request by WackerO**
Updated usage documentation, closes #158 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -414,6 +437,10 @@ Set to receive plain-text e-mails instead of HTML formatted.
 
 Set to disable colourful command line output and live life in monochrome.
 
+### `--multiqc`
+
+Path to the MultiQC zipped folder containing the MultiQC plots generated by the RNAseq pipeline, the MultiQC html report, and the MultiQC raw data.
+
```
Comment (ggabernet) [inline docs/usage.md:443]: This is not a core param, so you could move it to the section above with the rest of `rnadeseq` specific params.

Comment (ggabernet) [reply docs/usage.md:443]: At the moment it is optional, right?

**Review by ggabernet — APPROVED** [review]
Just one comment regarding the multiqc param, to move it above with the rest of rnadeseq specific params, otherwise LGTM!
Maybe also double check one more time that all params in `nextflow.config` are documented :)

File: docs/usage.md (hunk 1 above)
Comment (WackerO) [reply docs/usage.md]: Yes, I moved it as you suggested and added that it is an optional param (as well as the citest param, even though that is just for development)

**Pull request merged**